modules/RTC/TraceablePeerConnection.js

```diff
@@ -1318,6 +1318,43 @@ TraceablePeerConnection.prototype.createDataChannel = function(label, opts) {
 };
 
 /**
+ * Ensures that the simulcast ssrc-group appears after any other ssrc-groups
+ * in the SDP so that simulcast is properly activated.
+ *
+ * @param {Object} localSdp the WebRTC session description instance for
+ * the local description.
+ * @private
+ */
+TraceablePeerConnection.prototype._ensureSimulcastGroupIsLast
+= function(localSdp) {
+    let sdpStr = localSdp.sdp;
+
+    const videoStartIndex = sdpStr.indexOf('m=video');
+    const simStartIndex = sdpStr.indexOf('a=ssrc-group:SIM', videoStartIndex);
+    let otherStartIndex = sdpStr.lastIndexOf('a=ssrc-group');
+
+    if (simStartIndex === -1
+        || otherStartIndex === -1
+        || otherStartIndex === simStartIndex) {
+        return;
+    }
+
+    const simEndIndex = sdpStr.indexOf('\r\n', simStartIndex);
+    const simStr = sdpStr.substring(simStartIndex, simEndIndex + 2);
+
+    sdpStr = sdpStr.replace(simStr, '');
+    otherStartIndex = sdpStr.lastIndexOf('a=ssrc-group');
+    const otherEndIndex = sdpStr.indexOf('\r\n', otherStartIndex);
+    const sdpHead = sdpStr.slice(0, otherEndIndex);
+    const simStrTrimmed = simStr.trim();
```

Comment:
We can re-arrange the code so that we don't call trim(), but it is not equivalent to not copying it in the first place.

We would also have to change this `sdpStr = sdpStr.replace(simStr, '');` to this `sdpStr = sdpStr.replace(simStr + '\r\n', '');`.

Would you prefer this?